src/shared/config/paths.ts

```diff
@@ -0,0 +1,9 @@
+import ProfilePage from "pages/profile-page/ui/profile-page";
+
+export const paths = [
+  {
+    path: "/",
+    component: ProfilePage,
```

Comment:
oh, shared can't know about pages

Reply:
i see, have changed it to just obj with path and key